### ISSUE TITLE
WARNING: Don't be an idiot likeme and run this

### ISSUE THREAD

**Issue by evlo** [ionuttbara/melody_windows#17]
This thing does not ask for anything and runs all registry hacks etc. on start, there are no backups or reversts.

What I encountered in just a few minutes after install:
- dark theme everywhere
- most of the 2D HW acelarted stuff on windows is disabled, so laggy windows
- smooth scrolling disabled
- no search, nowhere, not in exlporer, not in start, not in settings, just no + windows search **service is not possible to start** in hopes of fixing this
- lot of apps loose settings, history is deleted for edge
- onedrive totally broken, **not even possible to edit office documents**
- changes cursor size to the largest (i guess)
- laggy keyboard
- touchpad palm rejection disabled, so it is almost impossible to type
- stuff missing from context menus and other things are added
- explorer view changed, missing recents, status bar etc.
- changed power/sleep button behaviour
- snipping tool does not open on print screen press
- shows seconds in time section of the taskbar (again, just a things I found in a first few minutes it broke, so I'm including this)

it might make sense to download "source" and look for registry hacks, as that is easirer then searching the web etc.

Basically what the correct description for this is:
one guys repo of windows settings he uses

Note: I'm not mad or angry at the author of this thing, I'm angry at myself. I just wanted to help other people who sumble on this thing. (And myself if I find this in the future)

**Comment by Syed123221** [comment]
It's scam

**Comment by evlo** [comment]
I did created PR to warn others https://github.com/evlo/melody_windows/tree/patch-1 as I can understand that from authors point of view there is no issue with this thing.

**Comment by evlo** [comment]
Well eve reinstallaing windows, in place does not fix the search not working, i guess lot of other thing will also not be fixed, so only real reainstall or iplace losing date (what is the point then) would fix it.
